### DISCUSSION TITLE
deploying to Google App Engine

### DISCUSSION THREAD

**Discussion by dotfrankruan** [dani-garcia/vaultwarden#2580] [Ideas]
Seems like it was written in Ruby so is it possible to deploy on Google App Engine since it provides a free tier.

**Comment by RealOrangeOne** [comment]
Vaultwarden is definitely not written in Ruby... It's Rust.

I'm not super familiar with App Engine, but if it either lets you [compile the binary yourself](https://github.com/dani-garcia/vaultwarden/wiki/Building-binary), or run a [docker container](https://github.com/dani-garcia/vaultwarden/wiki/Which-container-image-to-use), you should have absolutely no problems.
